### PR TITLE
django 1.8 compatibility fixes

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

File: jingo/__init__.py [hunk 1]
```diff
@@ -9,12 +9,17 @@
...
 from django.conf import settings
 from django.template.base import Origin, TemplateDoesNotExist
-from django.template.context import get_standard_processors
 from django.template.loader import BaseLoader
 from django.utils.importlib import import_module
 
 import jinja2
 
+try:
+    from django.template.engine import Engine
```
Comment (jsocol) [inline jingo/__init__.py]: can we just put the `get_standard_processors` definition here, so we don't need to do the `Engine = None` trick or worry about `import _ as ...`? I.e. something like this, except PEP8ed:

```python
try:
    from django.template.engine import Engine
    def get_standard_processors():
        return Engine.get_default().template_context_processors
except ImportError:
    from django.template.context import get_standard_processors
```

**Comment by eviljeff** [comment]
Sure, done.

**Comment by eviljeff** [comment]
Actually, after adding 1.8 to the travis/tox config some of the tests are failing so it might need more work than just this.  (I didn't get zamboni to a functional state on 1.8 - get_standard_processors was just an early failure I was able to easily fix)

**Comment by eviljeff** [comment]
Okay, added django1.8 to travis for python 2.7 and 3.3 and with a few more changes tests pass.  Note I've not tried these changes in a working django 1.8 project.

**Comment by jsocol** [comment]
There are a couple of outstanding issues here but in the interest of speed I'm just going to patch them up, since they're small.

**Pull request closed without merging**